admin/services/restart.service.js

```diff
@@ -122,8 +122,8 @@ restartService.getSubmittedRestarts = async schoolId => {
   if (!pupils || pupils.length === 0) return []
   let restarts = []
   // TODO: This loop is applied due to Cosmos MongoDB API bug and needs to be replaced with the new DB implementation
-  const latestPupilRestarts = await Promise.filter(pupils.map(async p => {
-    const restart = await pupilRestartDataService.findLatest({ pupilId: p._id, isDeleted: false })
+  const latestPupilRestarts = await bluebird.filter(pupils.map(async p => {
+    const restart = await pupilRestartDataService.sqlFindLatestRestart(p._id)
```

Comment:
If `p` here comes from mongo then it's fine.  Otherwise, you'll need `p.id`

Reply:
@jon-shipley updated